CarShowRoom/core/serializers.py

```diff
@@ -22,6 +24,8 @@ class RegisterSerializer(serializers.ModelSerializer):
     )
 
     def create(self, validated_data):
+        if hasattr(self, "Meta"):
+            validated_data["user_type"] = self.Meta.model.__name__.upper()
```

Comment:
what for?

Reply:
It is used to automatically fill user_type field in models, choices in user types and similar with models names so I used this way to fulfill this field